24.09.2020 2:16:38 : Дешифрование пароля успешно завершено
24.09.2020 2:29:12 : Дешифрование пароля успешно завершено
24.09.2020 2:29:18 : Хеш-последовательность успешно получена
24.09.2020 2:29:19 : Хеш-последовательность успешно получена
24.09.2020 2:45:09 : Дешифрование пароля успешно завершено
24.09.2020 2:52:27 : Дешифрование пароля успешно завершено
24.09.2020 2:52:36 : Хеш-последовательность успешно получена
24.09.2020 2:52:36 : Хеш-последовательность успешно получена
24.09.2020 2:52:36 : Дешифрование пароля успешно завершено
24.09.2020 4:04:27 : Дешифрование пароля успешно завершено
24.09.2020 4:04:32 : Хеш-последовательность успешно получена
24.09.2020 4:04:32 : Хеш-последовательность успешно получена
24.09.2020 4:04:32 : Дешифрование пароля успешно завершено
24.09.2020 4:06:04 : Дешифрование пароля успешно завершено
24.09.2020 4:06:10 : Хеш-последовательность успешно получена
24.09.2020 4:06:10 : Хеш-последовательность успешно получена
24.09.2020 4:06:10 : Дешифрование пароля успешно завершено
24.09.2020 4:09:37 : Дешифрование пароля успешно завершено
24.09.2020 4:09:44 : Хеш-последовательность успешно получена
24.09.2020 4:09:44 : Хеш-последовательность успешно получена
24.09.2020 4:09:44 : Дешифрование пароля успешно завершено
24.09.2020 4:11:47 : Дешифрование пароля успешно завершено
24.09.2020 4:12:05 : Дешифрование пароля успешно завершено
24.09.2020 4:12:10 : Хеш-последовательность успешно получена
24.09.2020 4:12:10 : Хеш-последовательность успешно получена
24.09.2020 4:12:10 : Дешифрование пароля успешно завершено
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:37 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:12:38 : Npgsql.PostgresException (0x80004005): 23503: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 942
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   в Npgsql.NpgsqlConnector.<>c__DisplayClass160_0.<<DoReadMessage>g__ReadMessageLong|0>d.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlConnector.cs:строка 973
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в System.Threading.Tasks.ValueTask`1.get_Result()
   в Npgsql.NpgsqlDataReader.<NextResult>d__44.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 449
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlDataReader.NextResult() в C:\projects\npgsql\src\Npgsql\NpgsqlDataReader.cs:строка 306
   в Npgsql.NpgsqlCommand.<ExecuteReaderAsync>d__102.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 1189
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.<ExecuteNonQuery>d__92.MoveNext() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 977
--- Конец трассировка стека из предыдущего расположения, где возникло исключение ---
   в System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   в System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   в Npgsql.NpgsqlCommand.ExecuteNonQuery() в C:\projects\npgsql\src\Npgsql\NpgsqlCommand.cs:строка 959
   в SZMK.Desktop.Services.Request.InsertAddDetail(Order Order, Detail Detail) в C:\Users\thedr\Desktop\SZMK-Software-Package\SZMK.Desktop\Services\Request.cs:строка 1617
  Exception data:
    Severity: ОШИБКА
    SqlState: 23503
    MessageText: INSERT или UPDATE в таблице "AddDetail" нарушает ограничение внешнего ключа "AddDetail_fk1"
    Detail: Ключ (OrderID)=(0) отсутствует в таблице "Orders".
    SchemaName: public
    TableName: AddDetail
    ConstraintName: AddDetail_fk1
    File: d:\pginstaller_12.auto\postgres.windows-x64\src\backend\utils\adt\ri_triggers.c
    Line: 2474
    Routine: ri_ReportViolation
24.09.2020 4:17:25 : Дешифрование пароля успешно завершено
24.09.2020 4:17:31 : Хеш-последовательность успешно получена
24.09.2020 4:17:31 : Хеш-последовательность успешно получена
24.09.2020 4:17:31 : Дешифрование пароля успешно завершено
24.09.2020 4:17:58 : Дешифрование пароля успешно завершено
24.09.2020 4:18:04 : Хеш-последовательность успешно получена
24.09.2020 4:18:04 : Хеш-последовательность успешно получена
24.09.2020 4:18:04 : Дешифрование пароля успешно завершено
24.09.2020 4:18:20 : Дешифрование пароля успешно завершено
24.09.2020 4:18:25 : Хеш-последовательность успешно получена
24.09.2020 4:18:25 : Хеш-последовательность успешно получена
24.09.2020 4:18:25 : Дешифрование пароля успешно завершено
24.09.2020 4:20:46 : Дешифрование пароля успешно завершено
24.09.2020 4:20:51 : Хеш-последовательность успешно получена
24.09.2020 4:20:51 : Хеш-последовательность успешно получена
24.09.2020 4:20:51 : Дешифрование пароля успешно завершено
